doc/theoretical_description_multilabel_classification.rst

```diff
@@ -175,14 +185,17 @@ hypothesis testing. We can express the goal of the procedure as follows:
 In order to find all the parameters :math:`\lambda` that satisfy the above condition, Learn Then Test propose to do the following:
 
 1: First across the collections of functions :math:`(T_\lambda)_{\lambda\in\Lambda}`, we estimate the risk on the calibration data
```

Comment:
These are not bullet points, not sure if that's the aim, but I don't find it very easy to read. Maybe put the numbers in bold?

Reply:
Done, adding bullet points

Reply:
Be careful some trouble in the documentation (indentation).